### PR TITLE
Topic/tweaks

### PR DESCRIPTION
**Pull request by lestrrat**

### CODE REVIEW

**Comment by syohex** [comment]
Looks good

File: filter_test.go [hunk 1]
```diff
@@ -21,10 +21,10 @@ func TestFuzzyFilter(t *testing.T) {
...
 		{"this is a test to Test the fuzzy filter", "TU", false}, // case sensitivity
-		{"日本語は難しいです", "難", true}, // kanji
-		{"あ、日本語は難しいですよ", "あい", true}, // hiragana
-		{"パソコンは遅いですネ", "ソネ", true}, // katakana
-		{"🚴🏻 abcd efgh", "🚴🏻e", true}, // unicode
+		{"日本語は難しいです", "難", true},                                 // kanji
+		{"あ、日本語は難しいですよ", "あい", true},                             // hiragana
+		{"パソコンは遅いですネ", "ソネ", true},                               // katakana
+		{"🚴🏻 abcd efgh", "🚴🏻e", true},                            // unicode
 	}
```
Comment (mattn) [inline filter_test.go]: please add test for empty string. and test matches first/last latter.

```
This is a test to Test the fuzzy filteR
```
then `TTR`

**Comment by mattn** [comment]
lol

**Comment by lestrrat** [comment]
ExternalCommandをガッちゃんこし忘れてたのでやりました

**Comment by syohex** [comment]
Latest change breaks `fuzzy` filter.

How to reproduce.
1. Launch peco like `% seq 100 | peco`. Input of `peco` should be multiple lines
2. Switch to fuzzy filter
3.  Input any characters
4. then panic

```
panic: runtime error: index out of range

goroutine 50 [running]:
panic(0x18d380, 0xc42000a100)
	/usr/local/Cellar/go/1.7.4/libexec/src/runtime/panic.go:500 +0x1a1
github.com/peco/peco.(*ListArea).Draw(0xc42007f000, 0xc420083500, 0x285f80, 0xc42008b4e0, 0x21, 0x0)
	/Users/shohei.yoshida/go/src/github.com/peco/peco/layout.go:524 +0xfab
github.com/peco/peco.(*BasicLayout).DrawScreen(0xc42008b4e0, 0xc420083500, 0x0)
	/Users/shohei.yoshida/go/src/github.com/peco/peco/layout.go:630 +0xc9
github.com/peco/peco.(*View).drawScreen(0xc42008b500, 0x285220, 0xc42010a160, 0x0)
	/Users/shohei.yoshida/go/src/github.com/peco/peco/view.go:86 +0x74
github.com/peco/peco.(*View).Loop(0xc42008b500, 0x285ca0, 0xc42007efc0, 0xc42008b440, 0x0, 0x0)
	/Users/shohei.yoshida/go/src/github.com/peco/peco/view.go:66 +0x4b8
created by github.com/peco/peco.(*Peco).Run
	/Users/shohei.yoshida/go/src/github.com/peco/peco/peco.go:317 +0x5f1
```

**Comment by lestrrat** [comment]
@syohex  ループって難しいですね！直しました

**Comment by syohex** [comment]
Looks good
